tephra-core/src/main/java/co/cask/tephra/persist/HDFSTransactionStateStorage.java

```diff
@@ -375,7 +379,7 @@ public static void main(String[] args) {
     Configuration config = new ConfigurationFactory().get();
 
     HDFSTransactionStateStorage storage =
-      new HDFSTransactionStateStorage(config, new SnapshotCodecProvider(config));
+      new HDFSTransactionStateStorage(config, new SnapshotCodecProvider(config), new TxMetricsCollector());
```

Comment:
Similarly to the TransactionStateCache case, this code path is only exercised when running the HDFSTransactionStateStorage class as a command line utility to inspect saved tx snapshots and log files.  So there are no relevant metrics to report here.

Again, I can add a comment here to indicate that use of the no-op collector is intentional.